packages/eas-cli/src/project/publish.ts

```diff
@@ -282,6 +294,7 @@ export async function uploadAssetsAsync(assetsForUpdateInfoGroup: CollectedAsset
     })
   );
 
+  updateSpinnerText?.(`${missingAssets.length} new assets being uploaded`);
```

Comment:
Thinking about this some more, the phrasing of "being uploaded" is a bit unwieldy compared to saying "uploading X new assets" but the issue with that is we prefix the line with "Uploading assets..." and "Uploading assets... uploading X new assets" is awkward too.

There also needs to be `${missingAssets.length === 1 ? 'asset' : 'assets'}` in there too.

Reply:
is there some npm `plural()` package 😄

Reply:
all of the awkwardness you mentioned is real. i prefer: `Uploading assets... 23 new assets uploading`. It keeps with this convention of (prefix) (number) (description)

Reply:
'Uploading assets... 23 new assets uploading" sounds fine to me